test/spec/directives/service-definition-edit.js

```diff
@@ -785,8 +785,8 @@ describe('Directive: serviceDefinitionEdit', function () {
         });
       });
 
-      ['links', 'external_links', 'ports', 'volumes', 'environment'].forEach(function (key) {
-        describe('when the key (' + key + ') represents a string value', function () {
+      ['links', 'external_links', 'ports', 'expose', 'volumes', 'volumes_from', 'environment', 'env_file', 'dns', 'cap_add', 'cap_drop', 'dns_search', 'labels'].forEach(function (key) {
+        describe('when the key (' + key + ') represents a sequence value', function () {
           it('returns an empty array', function () {
             var result = element.isolateScope().createNewEmptyValueForKey(key);
             expect(result).toEqual(['']);
```

Comment:
What would happen if this were actually an empty array?
`expect(result).toEqual([]);`